doc/api/cli.md

```diff
@@ -411,14 +411,18 @@ Node options that are allowed are:
 - `--enable-fips`
 - `--force-fips`
 - `--icu-data-dir`
+- `--inspect-brk`
```

Comment:
Useful when running node via a shell script, and wanting to cause node when it runs to break on first line. I'm not sure how much thought went into this initially. I can leave only the -port if that's preferred.